### ISSUE TITLE
Contract test failure in PhantomJS

### ISSUE THREAD

**Issue by chevdor** [ethereum/ethereum.js#106]
The following code is tested in test/eth.contract.js:
````
 abi.forEach(function (method) {
        if (method.name.indexOf('(') === -1) {
            var displayName = method.name;
            var typeName = method.inputs.map(function(i){return i.type; }).join();
            method.name = displayName + '(' + typeName + ')';
        }
    });
````

The 6 tests pass in the mocha tests, they succeed in Chrome, Safari and Firefox but fail in PhantomJS.
The 6 failures are similar to the following:
````
PhantomJS 1.9.8 (Mac OS X) contract should create simple contract with one method from abi with explicit type name FAILED
	TypeError: 'null' is not an object (evaluating 'abi.forEach')
	    at Contract (/var/folders/ff/pqx4w8zx5yd73j2vq4004r4c0000gn/T/8142a9386299b0e8a06e9b58fc7bf23f.browserify:1583:0 <- lib/web3/contract.js:225:0)
	    at /var/folders/ff/pqx4w8zx5yd73j2vq4004r4c0000gn/T/8142a9386299b0e8a06e9b58fc7bf23f.browserify:1117:0 <- lib/web3.js:52:0
	    at /var/folders/ff/pqx4w8zx5yd73j2vq4004r4c0000gn/T/8142a9386299b0e8a06e9b58fc7bf23f.browserify:30251:0 <- test/eth.contract.js:26:0
```

**Comment by chevdor** [comment]
Issue in phantomjs. I found a fix.

**Comment by chevdor** [comment]
Fix included in pull request #104

**Comment by chevdor** [comment]
I close this issue as the issue came from #104 and is also fixed in #104.
